README.md

```diff
@@ -46,6 +46,19 @@ bus.bind("order", {
 });
 ```
 
+## Configuration
+
+| Name                                     | Type  | Default               | Description                                              |
+| ---------------------------------------- | ----- | --------------------- | -------------------------------------------------------- |
+| **BUS_ADAPTER**                          | `str` | `kafka`               | RIPE Bus adapter (eg: `kafka`, `kafkaRetry`).            |
+| **KAFKA_HOSTS**                          | `str` | `localhost:9092`      | Kafka nodes available, separated by a `,` (comma).       |
+| **KAFKA_CLIENT_ID**                      | `str` | `ripe-kafka`          | Kafka client ID.                                         |
+| **KAFKA_CONSUMER_GROUP_ID**              | `str` | `ripe-kafka-consumer` | Kafka consumer ID.                                       |
```

Comment:
Please add a little more detail, so that someone who's not familiar to Kafka can get a grasp on what do these mean and if he should change them or not.